src/components/pages/GenerationPage/GenerationPage.js

```diff
@@ -187,7 +188,14 @@ function GenerationPage() {
                 </Button>
               </div>
               <div>
-                <Button type="button" onClick={() => generateAdviceLetter()}>
+                {/*
+                Legal team requested this be temporarily removed from UI
+                 */}
+                <Button
+                  type="button"
+                  className={cx('w-0', { invisible: true })}
+                  onClick={() => generateAdviceLetter()}
+                >
```

Comment:
`invisible: true` will make this button render in html and hidden in CSS. So theoretically the user can open up dev tools and change the CSS to `display: block;` or whatever. I'd rather just hide it for now by doing the following:

```suggestion
                {false && (
                    <Button
                      type="button"
                      className={cx('w-0', { invisible: true })}
                      onClick={() => generateAdviceLetter()}
                    >
               ) }
```

If our overzealous linter complains about the above, you can just comment it out instead